src/robusta/cli/main.py

```diff
@@ -288,6 +288,13 @@ def gen_config(
             },
         ]
 
+    if is_small_cluster:
+        setattr(values, "prometheus", {})
```

Comment:
I think this should be under a `kube-prometheus-stack` section

Reply:
fixed